### ISSUE TITLE
YAM 2.10-dev (30-09-2014) shows GR on TCP/IP prefs.

### ISSUE THREAD

**Issue by jens-maus** [jens-maus/yam#586]
**:bust_in_silhouette: Originally by _javierdlr@euskalnet.net_ on 2014-09-30 11:03:42 +0200**
<hr>
## Summary
Trying to go to TCP/IP settings shows a GR, with laetst YAM 2.10-dev (30-09-2014)

## Steps to reproduce
 1. Open Prefs window
 2.Click on TCP/IP page

## Expected results
See TCP/IP options
## Actual results
GR shows up
## Notes

**Comment by jens-maus** [comment]
**:bust_in_silhouette: Originally by @tboeckel on 2014-09-30 11:30:09 +0200**
<hr>
In "8269":
```
#CommitTicketReference repository="" revision="8269"
* mui/misc: the state variable for the NextObject() call must be initialized with the list's lh_Head variable instead of the result of the GetHead() function, because NextObject() does not check for NULL pointers. This closes #586.
```

**Comment by jens-maus** [comment]
**:bust_in_silhouette: Originally by _javierdlr@euskalnet.net_ on 2014-10-01 11:08:10 +0200**
<hr>
Confirmed, now TCP/IP page opnes without GR. :-)
THX
